phpstan.json

```diff
@@ -1,3 +1,3 @@
 {
-  "defaultLevel": 5
+  "defaultLevel": 6
```

Comment:
why 6, if checked with 8 as per stan script?

Reply:
Raised phpstan default level to 8